lua/telescope/mappings.lua

```diff
@@ -75,7 +77,7 @@ local get_next_id = function()
   return _mapping_key_id
 end
 
-local assign_function = function(prompt_bufnr, func)
+local assign_function = function(prompt_bufnr, func, mode, keybind)
```

Comment:
we no longer need to accept mode, keybind here. Also no longer need to pass it in down below